Godeps/Godeps.json

```diff
@@ -435,82 +462,82 @@
 		},
 		{
 			"ImportPath": "github.com/vmware/govmomi",
-			"Comment": "v0.1.0-454-g9051bd6",
+			"Comment": "v0.6.2",
```

Comment:
Ah, weird... Ok thanks.  I don't think there's anything we can do about that WRT this change so I guess we'll just have to accept that weirdness for the version bump.